controllers/comments/index.js

```diff
@@ -23,7 +23,7 @@ export default {
       });
     } catch (e) {
       /* istanbul ignore next */
-      return res.status(500).json({
+      return res.status(500).send({
         message: 'something went wrong'
```

Comment:
Good job, kindly correct the `message` key to `error` as agreed by the team.

Reply:
Thank you.